OpenRA.Mods.Common/Traits/Buildings/Building.cs

```diff
@@ -154,7 +154,8 @@ public WVec CenterOffset(World w)
 		public Actor FindBaseProvider(World world, Player p, CPos topLeft)
 		{
 			var center = world.Map.CenterOfCell(topLeft) + CenterOffset(world);
-			var allyBuildEnabled = world.WorldActor.Trait<MapBuildRadius>().AllyBuildRadiusEnabled;
+			var mapBuildRadius = world.WorldActor.Trait<MapBuildRadius>();
+			var allyBuildEnabled = mapBuildRadius.AllyBuildRadiusEnabled;
```

Comment:
Please add an 
```csharp
if (!mapBuildRadius)
	return null;
```
as there are no base providers if the build radius is disabled.

Reply:
where exactly?

Reply:
will do